### PR TITLE
54 store create UI menu

### PR DESCRIPTION
**Pull request by quyn2904**
image

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -1,2 +1 @@
-npm run lint-staged
-npm run build
```
Comment (minhhy2801) [inline .husky/pre-commit:2]: why you remve it?

File: .prettierrc.json [hunk 2]
```diff
@@ -7,5 +7,5 @@
   "endOfLine": "auto",
   "useTabs": false,
   "singleQuote": false,
-  "printWidth": 80
+  "printWidth": 120
```
Comment (minhhy2801) [inline .prettierrc.json:10]: why do you change it?

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (minhhy2801) [inline .vscode/settings.json:1]: I don't think you should push this file

File: tailwind.config.ts [hunk 4]
```diff
@@ -4,11 +4,7 @@ import { nextui } from "@nextui-org/react";
 import { Config } from "tailwindcss/types/config";
 
 const config: Config = {
-  content: [
-    "./index.html",
-    "./src/**/*.{js,ts,jsx,tsx}",
-    "./node_modules/@nextui-org/theme/dist/**/*.{js,ts,jsx,tsx}",
-  ],
+  content: ["./index.html", "./src/**/*.{js,ts,jsx,tsx}", "./node_modules/@nextui-org/theme/dist/**/*.{js,ts,jsx,tsx}"],
```
Comment (minhhy2801) [inline tailwind.config.ts:7]: You missing format now

File: src/pages/VerifyAccount/VerifyAccount.tsx [hunk 5]
```diff
@@ -249,22 +196,13 @@ const VerifyAccount = () => {
...
           )}
 
-          <Button
-            className="w-full mt-12 text-lg"
-            size="lg"
-            color="primary"
-            variant="ghost"
-            onClick={handleVerifyOTP}
-          >
+          <Button className="w-full mt-12 text-lg" size="lg" color="primary" variant="ghost" onClick={handleVerifyOTP}>
```
Comment (minhhy2801) [inline src/pages/VerifyAccount/VerifyAccount.tsx:205]: The previous format is good, please revert that

File: src/components/AdvertiseBar.tsx [hunk 6]
```diff
@@ -0,0 +1,8 @@
+const AdvertiseBar = () => {
```
Comment (minhhy2801) [inline src/components/AdvertiseBar.tsx:1]: I don't think this should be a component

Comment (quyn2904) [reply src/components/AdvertiseBar.tsx:1]: image
I create this component for later, as it is independent from navbar and just appear on several pages

**Pull request merged**